storage/retry_conformance_test.go

```diff
@@ -521,34 +523,42 @@ func TestRetryConformance(t *testing.T) {
 						t.Logf("No tests for operation %v", methodName)
 					}
 					for i, fn := range methods[methodName] {
-						testName := fmt.Sprintf("%v-%v-%v-%v", retryTest.Id, instructions.Instructions, methodName, i)
-						t.Run(testName, func(t *testing.T) {
-
-							// Create the retry subtest
-							subtest := &emulatorTest{T: t, name: testName, host: endpoint}
-							subtest.create(map[string][]string{
-								method.Name: instructions.Instructions,
+						transports := []string{"json", "grpc"}
+						for _, transport := range transports {
+							testName := fmt.Sprintf("%v-%v-%v-%v/%v", retryTest.Id, instructions.Instructions, methodName, i, transport)
```

Comment:
Maybe start with the transport? Seems most relevant to filter by

Reply:
updated